controllers/result.go

```diff
@@ -16,6 +19,8 @@ type ResultController struct {
 	BaseController
 }
 
+const ITEMS_PER_PAGE = 20
```

Comment:
How about moving this to the application config? I think this value should be configurable.

Reply:
Yeah, it's a great idea.  ✅  moved this to the configuration on [57ca4d5](https://github.com/carryall/go-google-scraper-challenge/pull/68/commits/57ca4d516aa2ad2b6fcb0da2c2e19f5b1418d592)